tests/integration_tests/cmd/test_status.py

```diff
@@ -14,7 +14,11 @@ def _wait_for_cloud_init(client: IntegrationInstance):
     for _ in range(30):
         try:
             result = client.execute("cloud-init status --long")
-            if result and result.ok:
+            if (
+                result
+                and result.ok
+                and ("running" not in result or "not run" not in result)
```

Comment:
Is there a reason we need `--long` here?

I don't think we are checking the extra output so to avoid future false positives ("running" or "not run" in the extra details) think we should drop it?